### PR TITLE
Componente de modal creado

### PR DESCRIPTION
**Pull request by juasmartinezbelS4N**
## Opción única

Captura de Pantalla 2020-06-04 a la(s) 6 16 20 p  m

Captura de Pantalla 2020-06-04 a la(s) 6 16 27 p  m

Captura de Pantalla 2020-06-04 a la(s) 6 17 01 p  m

## Dos opciones

Captura de Pantalla 2020-06-04 a la(s) 6 19 12 p  m
Captura de Pantalla 2020-06-04 a la(s) 6 19 08 p  m
Captura de Pantalla 2020-06-04 a la(s) 6 18 53 p  m

## Sin Botón Cerrar

Captura de Pantalla 2020-06-04 a la(s) 6 19 50 p  m

## Sin opciones

Captura de Pantalla 2020-06-04 a la(s) 6 35 42 p  m

### Overflow de 3 Lineas único

Captura de Pantalla 2020-06-04 a la(s) 6 20 21 p  m

Captura de Pantalla 2020-06-04 a la(s) 6 20 29 p  m

### Overflow de 3 Lineas duo

Captura de Pantalla 2020-06-04 a la(s) 6 21 03 p  m

Captura de Pantalla 2020-06-04 a la(s) 6 20 52 p  m

### Dos Columnas

Captura de Pantalla 2020-06-04 a la(s) 6 21 48 p  m
Captura de Pantalla 2020-06-04 a la(s) 6 22 03 p  m

### Mucho contenido

ezgif-3-e730f07487a8

### CODE REVIEW

File: src/components/modal/modalCo.js [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+    textAlign: "justify",
+    textOverflow: "ellipsis",
+  },
+  /**
+   * Variables to make the CloseSVG button stay in place
+   */
+  CloseSVG: {
+    float: "right",
+    position: "sticky",
+    top: "5vh",
```
Comment (setohe0909) [inline src/components/modal/modalCo.js]: why did you use vh?

File: src/components/modal/modalCo.js [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+    onPrimary,
+    onSecondary,
+  } = props
+  const currentProps = { ...defaultProps, ...props }
+
+  /**
+   * Sets the action portion of the modal, according to the props
+   */
+  let modalButton
+  if (primarybutton !== "") {
```
Comment (setohe0909) [inline src/components/modal/modalCo.js]: what do you think about using the following statement.. 

````
 
if(!primarybutton){
........
````

The above case you validate empty and undefined values

File: src/components/modal/modalColumsCo.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+function ModalColumsCo(props) {
+  const { children } = props
+  const styles = useStyles()
+  let child1
+  let child2
+  /**
+   * Divides the children in order to place them properly
+   */
+  if (Array.isArray(children)) {
+    ;[child1, child2] = children
```
Comment (setohe0909) [inline src/components/modal/modalColumsCo.js:52]: I feel confused... what mean `;` 🤔

**Review by setohe0909 — COMMENTED** [review]
I left minor comments and a doubt

File: src/components/modal/modalCo.js (hunk 1 above)
Comment (juasmartinezbelS4N) [reply src/components/modal/modalCo.js]: Recommendation from some coworkers, should I change it?

File: src/components/modal/modalColumsCo.js (hunk 3 above)
Comment (juasmartinezbelS4N) [reply src/components/modal/modalColumsCo.js:52]: I don't really know, prettier wants it
Captura de Pantalla 2020-06-05 a la(s) 6 42 22 a  m

File: src/components/modal/modalCo.js (hunk 2 above)
Comment (juasmartinezbelS4N) [reply src/components/modal/modalCo.js]: Sometimes I forget about that, thanks!

**Pull request merged**